### PR TITLE
DOC: Add status badges to `README` file

### PR DESCRIPTION
**Pull request by jhlegarreta**
- DOC: Add status badges to `README` file
- DOC: Make `README` refs be considered as citations instead of footnotes
- DOC: Use URL as path to static content in `README`

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Doc failure is unrelated:
https://github.com/nipreps/eddymotion/actions/runs/8376155506/job/22935185353?pr=138#step:6:54

File: README.rst [hunk 1]
```diff
@@ -6,6 +6,22 @@
    :target: https://doi.org/10.5281/zenodo.4680599
    :alt: DOI
 
+.. image:: https://img.shields.io/pypi/l/mriqc.svg
+   :target: https://pypi.python.org/pypi/mriqc/
```
Comment (oesteban) [inline README.rst:10]:
```suggestion
.. image:: https://img.shields.io/pypi/l/eddymotion.svg
   :target: https://pypi.python.org/pypi/eddymotion/
```

Comment (jhlegarreta) [reply README.rst]: My bad. Changed in 4a66668.

**Comment by jhlegarreta** [comment]
Doc failure is again unrelated: cross-ref https://github.com/nipreps/eddymotion/pull/138#issuecomment-2012456967

**Comment by jhlegarreta** [comment]
https://github.com/nipreps/eddymotion/pull/138#issuecomment-2015235736 fixed in 6f8bb9a.

**Pull request merged**